### ISSUE TITLE
Screen resolution at iOS, returns "points" instead of "pixels"

### ISSUE THREAD

**Issue by Szymaniuk** [aritchie/deviceinfo#18]
Using:

```
Acr.DeviceInfo.DeviceInfo.Hardware.ScreenHeight;
Acr.DeviceInfo.DeviceInfo.Hardware.ScreenWidth;
```

we got **points** of screen, instead of **pixels** (real resolution), as it uses native:

```
UIScreen.MainScreen.Bounds.Height
UIScreen.MainScreen.Bounds.Width
```

behind.

What I found, is that it's just needed to multiply those values, with:

```
UIScreen.MainScreen.Scale
```

property, to get real screen (**pixels**) resolution.

Example in case of iPhone 6s, your plugin currently returns:

```
UIScreen.MainScreen.Bounds.Height: 667
UIScreen.MainScreen.Bounds.Width: 375
```

Iv'e just debbuged, that "scale" of this device is:

```
UIScreen.MainScreen.Scale: 2
```

so multipling 667 * 2 gives **1334**, and multiplying 375 * 2 gives **750**, which is real screen (**pixels**) resolution.

More info:
https://www.paintcodeapp.com/news/iphone-6-screens-demystified
https://forums.xamarin.com/discussion/comment/21938#Comment_21938
https://forums.xamarin.com/discussion/comment/58870#Comment_58870
https://forums.xamarin.com/discussion/comment/91294#Comment_91294
In short - scale will be from **1** to **3**, depends on Apple device.

Could you please enclose these changes in your plugin, byt multiplying those values, or by exposing **Scale** property? Better would be to just return real **pixels** resolution from my PoV.

Thanks a lot!

**Comment by gazeciarz** [comment]
+1

**Comment by adamzareba** [comment]
+1

**Comment by ikillforfood** [comment]
+1

**Comment by aritchie** [comment]
+1's are not welcome here... ty
